frontend/cypress/fixtures/copyDataset.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Move this to the existing `datasets.json` file and give the dataset a unique name/ID to identify it for your tests.